### ISSUE TITLE
Flacoco integration with SequenceR

### ISSUE THREAD

**Issue by andre15silva** [eclipse/repairnator#1239]
As far as I understood, GZoltar is used through Astor (https://github.com/eclipse/repairnator/blob/master/src/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/repair/sequencer/detection/AstorDetectionStrategy.java) in the sequenceR mode. 

This means that, once https://github.com/SpoonLabs/astor/pull/323 is merged, it will automatically use flacoco, as it will become the default fault localization mode.

**Comment by monperrus** [comment]
Nice.

**Comment by monperrus** [comment]
FTR https://github.com/SpoonLabs/astor/pull/323 is merged

**Comment by monperrus** [comment]
Do we need to push a new version of Sequencer image on Dockerhub?

**Comment by andre15silva** [comment]
Yes, but before we need to release a new version of https://github.com/SpoonLabs/astor and a new version of repairnator.
